sdks/java/core/src/main/java/org/apache/beam/sdk/util/HistogramData.java

```diff
@@ -58,6 +61,8 @@ public HistogramData(BucketType bucketType) {
     this.numBoundedBucketRecords = 0;
     this.numTopRecords = 0;
     this.numBottomRecords = 0;
+    this.mean = 0;
+    this.sum_of_squared_deviations = 0;
```

Comment:
for java, camel case is preferred https://google.github.io/styleguide/javaguide.html#s5.3-camel-case